Clover/app/src/main/java/org/floens/chan/ui/cell/ThreadStatusCell.java

```diff
@@ -147,6 +148,20 @@ public boolean update() {
                         String ips = op.getUniqueIps() + "P";
                         finalText = TextUtils.concat(finalText, " / " + ips);
                     }
+
+                    Page p = callback.getPage(op);
+                    String page = getContext().getString(R.string.thread_page_no) + " ";
+                    SpannableString pageS;
+                    if (p != null) {
+                        page = page + String.valueOf(p.page);
+                        pageS = new SpannableString(page);
+                        if (p.page >= board.pages) {
+                            pageS.setSpan(new StyleSpan(Typeface.ITALIC), 0, page.length(), 0);
+                        }
+                    } else {
+                        pageS = new SpannableString(page + "?");
+                    }
+                    finalText = TextUtils.concat(finalText, " / ", pageS);
```

Comment:
I know this wasn't you, but it might be good to refactor to build `result` using (`SpannableStringBuilder`)[https://developer.android.com/reference/android/text/SpannableStringBuilder.html]

Reply:
I can do that.